cache/lru_cache_test.cc

```diff
@@ -226,8 +226,7 @@ class FastLRUCacheTest : public testing::Test {
     cache_ = reinterpret_cast<fast_lru_cache::LRUCacheShard*>(
         port::cacheline_aligned_alloc(sizeof(fast_lru_cache::LRUCacheShard)));
     new (cache_) fast_lru_cache::LRUCacheShard(
-        capacity, false /*strict_capcity_limit*/, kDontChargeCacheMetadata,
-        24 /*max_upper_hash_bits*/);
+        capacity, 1, false /*strict_capcity_limit*/, kDontChargeCacheMetadata);
```

Comment:
When passing constants as parameters without obvious meaning (lots of parameters), it is encouraged to use a comment to indicate its meaning, such as `false /*strict_capacity_limit*/` here.